pyproject.toml

```diff
@@ -93,6 +93,7 @@ pydata-sphinx-theme = "*"
 pydocstyle = "*"
 pytest = "*"
 pytest-cov = "*"
+pytest-xdist = "^2.5.0"
```

Comment:
```suggestion
pytest-xdist = "*"
```

Reply:
We will need it in the two other places (one above, one below).

Reply:
Corrected in e2d0255c48e33c873a459fbfa1574659d89df3a5